### PR TITLE
avoid computing extents at all and just define as lonlat, use 512 blo…

### PR DESCRIPTION
**Pull request by AndrewAnnex**
…ck sizes to align with rio-cogeo

I think transform bounds was having issues so dropped it all together and got more realistic tiles coordinates, I think, for most projections. Will need to do something else for the polar projections

### CODE REVIEW

File: planetcantile/data/generate.py [hunk 1]
```diff
@@ -50,10 +50,10 @@ class Tmsparam(object):
     extent: tuple[float, float, float, float]
     # Tile Matrix Set coordinate reference system
     crs: CRS
-    # Width of each tile of this tile matrix in pixels (default is 256).
-    tile_width: int = 256
-    # Height of each tile of this tile matrix in pixels (default is 256).
-    tile_height: int = 256
+    # Width of each tile of this tile matrix in pixels (default is 512).
+    tile_width: int = 512
```
Comment (drferling) [inline planetcantile/data/generate.py:54]: Why the swap here? To match something else?

Comment (AndrewAnnex) [reply planetcantile/data/generate.py:54]: yes, rio-cogeo uses 512x512 tiles which is also the default in the COG GDAL driver (https://gdal.org/drivers/raster/cog.html), there is some ongoing discussion in #6 related to this. The good news for some is that the larger tiles would reduce the number network requests by a factor of 4 per zoom level.

Comment (AndrewAnnex) [reply planetcantile/data/generate.py:54]: also glad to see others are interested in these products @drferling, none of these tms specs are finalized yet so hopefully you aren't using them!

Comment (drferling) [reply planetcantile/data/generate.py:54]: :+1: I just pulled my notes up for doing a Titiler deploy in our AWS environment, so I'm not using these yet. My notes are reminding me that the deploy was a pain (normal permissions friction, not a struggle due to the code base). 

I am hoping to get this all deployed shortly, though, so hopefully using these quite soon.

**Review by drferling — APPROVED** [review]
Reviewed the generate.py. LGTM.

**Comment by AndrewAnnex** [comment]
seems to be working with poles correctly now. Screen shots are using the grids for 49930 and 49935 (north and south polar stereographic projections) to make a north and south image from MOLA. The first image has the project CRS set to 49935 and the second is set to 49930. The images look low res primarily because I first downsampled the data to 10km pixels and then only use the first zoom level in generating the cogs. See attached zip file if curious

that said this should still be validated a bit more. it's weird to me that the axis of the prime meridian sort of flips to being above or below the center of the image depending on the projection... but things are at least self consistent.

image
image
[mola_poles_proofofconcept.zip](https://github.com/AndrewAnnex/planetcantile/files/10575426/mola_poles_proofofconcept.zip)

**Pull request merged**